spec/unit/classes/server/service_spec.rb

```diff
@@ -0,0 +1,32 @@
+
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe 'postgresql::server::service', type: :class do
+
+  let :pre_condition do
+    'include postgresql::server'
+  end
+
+  let :node_params do
+    {
+      'lsbmajdistrelease' => '18.04',
+    }
+  end
+
+  let :facts do
+    {
+      os: {
+        family: 'Debian',
+        name: 'Ubuntu',
+        release: { 'full' => '18.04', 'major' => '18.04' },
+      },
+    }
+  end
+
+  it {
+    is_expected.to contain_class('postgresql::server::service')
+    is_expected.to contain_service('postgresqld').with_name('postgresql')
```

Comment:
This looks like it would also pass prior to this change which means it isn't a good test.

Reply:
I added service status to the test

Reply:
Could you chain it so there's a single `contain_service('postgresqld')`? I also think it's a best practice to use a single `is_expected.to` within `it {}`.